prow/gerrit/adapter/adapter.go

```diff
@@ -470,6 +493,10 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 				break
 			}
 		}
+		// Suspect that inrepoconfig takes long time, add a log for its duration
+		logger.WithField("duration(s)", time.Since(tracker).Seconds()).Debug("Time taken for founding jobs for gerrit change.")
+		tracker = time.Now()
```

Comment:
Why do we reset the tracker here, and on line 543? Did you mean to use it later to log another duration?

Reply:
the idea is to measure the time taken for each step, and if it's desired to do aggregated duration we could potentially sum the durations.